3scale/porta
# THREESCALE-10915: Fix Quick starts images in production mode

**What this PR does / why we need it**:

In production mode Quick starts images are not showing: 
![Screenshot from 2024-05-17 11-46-07](https://github.com/3scale/porta/assets/1270328/beb05249-fb7b-46c3-bf85-681b1f8b2330)

This is an alternative (and simpler) to the fix proposed in https://github.com/3scale/porta/pull/3729

The main issue is that the images were passing through the assets pipeline, and were added to the public assets folder with the fingerprint suffix, like this:
```
/public/assets/quickstarts/learn-about-products-afa799f1e9d67800ea82cdb113e2cf11e624d331f36818ab496c3a3f6d9f8934.png
```
However, there is no obvious way to reference these images from the Quick start's YAML definition, as we don't have direct access to `image_tag` helper (or at least I couldn't find a way...).

As these images are only used from Quick starts, I thought it was fine to just copy them as static assets in the `public` folder.

Maybe there is some better way, but I don't know it :) 

The PR also removes some duplicate images from the repo.

There is also this issue on the Quickstarts repo that seems relevant: https://github.com/patternfly/patternfly-quickstarts/issues/207

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-10915

**Verification steps** 

- configure porta for running in production mode
- precompile assets
- run Rails server in production mode
- verify that the images are visible in Quickstarts

**Special notes for your reviewer**:
